doc/changelog.rst

```diff
@@ -3,6 +3,147 @@
 Changelog
 =========
 
+.. _changelog-v2-10-0:
+
+v2.10.0
+-------
+Highlights
+
+TODO: add highlights
+
+This release includes
+
+Backwards incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Remove the deprecated option ``use_legacy_supplementaries`` (`#2202 <https://github.com/ESMValGroup/ESMValCore/pull/2202>`__) `Bouwe Andela <https://github.com/bouweandela>`__
+
+   - The recommended upgrade procedure is to remove ``use_legacy_supplementaries`` from config-user.yml
+     (if it was there) and remove any mention of fx_variables from the recipe. If automatically defining
+     the required supplementary variables does not work, define them in the variable or
+     (``additional_``) ``datasets`` section as described in :ref:`supplementary_variables`.
+
+-  Use smarter (units-aware) weights (`#2139 <https://github.com/ESMValGroup/ESMValCore/pull/2139>`__) `Manuel Schlund <https://github.com/schlunma>`__
+   
+   - Some preprocessors handle units better. For details, see the issue.
```

Comment:
Which issue?

Reply:
Sorry, that should be PR, meaning the PR linked just above. Do you think changing `..., see the issue.` to `..., see the PR.` is enough?

Reply:
Maybe spell it out as `pull request` for people who don't know what PR means?